hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveSessionTest.java

```diff
@@ -144,6 +144,59 @@ public void reactivePersist(TestContext context) {
 		);
 	}
 
+	@Test
+	public void reactivePersistInTx(TestContext context) {
```

Comment:
Interesting...  how come this is not called `reactiveTransactPersist`? :)

Reply:
Because word order: it's `persistInTransaction` not "reactiveInTransactionPersist`.